e2e/test/scenarios/permissions/admin-permissions.cy.spec.js

```diff
@@ -642,34 +642,90 @@ describe("scenarios > admin > permissions", () => {
     cy.signInAsAdmin();
   });
 
-  it("partial data permission updates should not remove permissions from other unmodified groups", () => {
-    // check the we have an expected initial state
-    cy.visit(`admin/permissions/data/group/${DATA_GROUP}`);
-    assertPermissionTable([["Sample Database", "Query builder and native"]]);
-
-    // make a change to the permissions of another group
-    selectSidebarItem("nosql");
-    assertPermissionTable([["Sample Database", "Query builder only"]]);
-    modifyPermission("Sample Database", NATIVE_QUERIES_PERMISSION_INDEX, "No");
-
-    // observe the save change request and assert that we don't get back
-    // values for groups we did not modify
-    cy.intercept("PUT", "/api/permissions/graph").as("updateGraph");
-
-    // save changes
-    cy.button("Save changes").click();
-    modal().within(() => {
-      cy.button("Yes").click();
-    });
+  context("partial updates", () => {
+    it("partial data permission updates should not remove permissions from other unmodified groups", () => {
```

Comment:
just moved this test block into this context, no changes within this `it`. the next `it` needs review though.